frontend/src/metabase/App.tsx

```diff
@@ -93,25 +98,43 @@ function App({
   children,
 }: AppProps) {
   const [viewportElement, setViewportElement] = useState<HTMLElement | null>();
-  const [errorInfo, setErrorInfo] = useState<ErrorInfo | null>(null);
+  const [errorInfo, setErrorInfo] = useState<(ErrorInfo & Error) | null>(null);
+  const [errorCount, setErrorCount] = useState(0);
+
+  const countError = () => setErrorCount(prev => prev + 1);
+  console.log("errorInfo", errorInfo);
```

Comment:
Not totally sure if the `console.log` here, is intentional. I guess not?

Reply:
🤦  thanks